app/controllers/ApplicationController.java

```diff
@@ -23,16 +19,12 @@
 public final class ApplicationController extends SunriseController {
 
     @Inject
-    public ApplicationController(final PlayJavaSphereClient client, final CategoryTree categoryTree, final Configuration configuration) {
-        super(client, categoryTree, configuration);
+    public ApplicationController(final PlayJavaSphereClient client, final CategoryTree categoryTree,
```

Comment:
Interesting, I've created a ticket https://github.com/sphereio/sphere-sunrise/issues/100 to apply it in the next PR, which by the way, has the honor to be the 100th issue ;)